app/Models/Concerns/HasFilters.php

```diff
@@ -111,6 +112,14 @@ public function scopeFilter(Builder $query, array $params = []): Builder
 
                 if ($key === 'tags') {
                     $this->filterTags($query, $value);
+                } elseif ($key == 'locations') {
+                    $this->filterMultipleLocations($query, $value);
```

Comment:
Would make more sense to rename filterLocations to filterLocation, and use filterLocations for the new one